.github/actions/templates/validateModuleDeployment/action.yml

```diff
@@ -130,12 +144,17 @@ runs:
 
         Test-TemplateWithParameterFile @functionInput -Verbose
 
+        Write-Output "::endgroup::"
+
     # [Deployment execution] task(s)
     # ------------------------------
     - name: 'Deploy [${{ inputs.templateFilePath }}] with parameters [${{ inputs.parameterFilePath }}]'
       shell: pwsh
       id: deploy_step
       run: |
+        # Deploy [${{ inputs.templateFilePath }}] with parameters [${{ inputs.parameterFilePath }}]
```

Comment:
Why a comment with tokens. It won't be displayed anyways. Should we maybe just have a generic `Grouping task` comment?

Reply:
Some for the other added comments